posts/2024-08-07-Project-goals.md

```diff
@@ -0,0 +1,89 @@
+
+## How to track progress
+
+Each of the proposed project goals has a dedicated tracking issue on the [rust-lang/rust-project-goals][] repository; these tracking issues are collected in the [2024h2 milestone][]; the assignee of those issues indicates the [owner][] for that particular work. The issues will receive regular status updates from their respective owners, and we will collate these updates into public blog posts.
+
+[rust-lang/rust-project-goals]: https://rust-lang.github.io/rust-project-goals/
+[2024h2 milestone]: https://github.com/rust-lang/rust-project-goals/milestone/2
+[owner]: https://rust-lang.github.io/rust-project-goals/about/owners.html
+
+It's worth stating up front: **we don't expect all of these goals to be completed**. Many of them were proposed and owned by volunteers, and it's normal and expected that things don't always work out as planned. In the event that a goal seems to stall out, we can either look for a new owner or just consider the goal again in the next round of goal planning.
+
+## How we selected project goals
+
+This **project goal** process started as an experiment, proposed in [RFC #3614][], to be run with nikomatsakis (myself) as the overall owner. Now that we've done it once, my intent is to author a follow-up RFC that describes the process we ended up with and codifies it for the future.
+
+The project goal process began with a [public call for proposals in early May][pcp]. Each proposal was phrased as a Pull Request (PR) against the [rust-lang/rust-project-goals][] repository. Goal proposal PRs follow a [goal template][] that describes the motivation and a high level sketch of the work that is planned for 2024; it's kind of like the "first half" of an RFC.
+
+[RFC #3614]: https://github.com/rust-lang/rfcs/pull/3614
+[pcp]: https://blog.rust-lang.org/inside-rust/2024/05/07/announcing-project-goals.html
+[goal template]: https://rust-lang.github.io/rust-project-goals/TEMPLATE.html
+
+Each goal represents a contract between the **owner** -- the person who will ensure the work gets done, and typically the person opening the PR -- and the **Rust team**. The owner is promising to put their own resources (time, money, expertise) into driving the work forward, and the team is giving their approval and commiting to support the owner in the ways requested (e.g., reviews, meetings, etc).
+
+The details of who precisely will do what are giving in the ["Ownership and team asks"][] section, which takes the place of the "detailed design" you might find in an RFC. In some cases, the only ask of the team is "discussion and moral support", meaning that the goal owner is mostly looking for official approval that this is something the team would like to see get done. But other goals included asks like "standard reviews" for the resulting implementation work, pre-allocated "design meetings" for the team to review the intended design and give feedback, or "RFC decision" if an RFC will be required. 
```

Comment:
I think this is meant for the main blog, right? In that case, these last two paragraphs may be a bit too detailed for readers that may not follow/know internal processes. But I think there is not going to be an internals post in the end, so I imagine that is probably why you ended up putting it here...

Reply:
Hmm... the next section may be more interesting too for some of that audience than the details at the end of this one. Perhaps reordering could be an option to get the most important bits out first.

For instance, the "Each goal represents a contract between the **owner**..." paragraph here sounds like it could go outside (before) this section, since it is important context outside of how the goals were selected, which then could perhaps allow you to move the rest of this section with details lower.

Reply:
I agree, it feels too "in the weeds" to me.